test/wdio.conf.js

```diff
@@ -32,7 +32,8 @@ exports.config = {
     // capabilities: https://docs.saucelabs.com/reference/platforms-configurator
     //
     capabilities: [{
-        browserName: 'chrome'
+        browserName: 'chrome',
+        maxDuration: 10800
```

Comment:
Why do we need this to be so long? It's far longer than any of our test sessions should be running.